### PR TITLE
CS-1848 Add copyright notice to `README.md`

### PR DESCRIPTION
**Pull request by sedwards2009**
CS-1848

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,6 @@
         "url": "git+https://github.com/Ultimaker/react-web-components.git"
     },
     "author": "Ultimaker",
-    "license": "ISC",
```
Comment (ChrisTerBeke) [inline package.json]: this repo is (deprectated but) open source, so we'll need to keep a license agreement in place.

Comment (sedwards2009) [reply package.json]: We might just have to leave this as-is, even though we don't have a copy of the license or any other mention elsewhere, just these three letters.

I don't think the license question was properly asked when this repo was set up either.

Comment (sedwards2009) [reply package.json]: Also, personally I would prefer to use MIT here, given that it is far more common while being functionally the same.

Comment (ianpaschal) [reply package.json]: ^Agree

**Review by ianpaschal — APPROVED** [review]

**Comment by sedwards2009** [comment]
This repo is obsolete anyway and having build troubles. closing

**Pull request closed without merging**